package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "mv_simple_react_calculator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "prop-types": "^15.7.2",
```

Comment:
We need to add `big.js`, seems you miss that 😄 
![image](https://user-images.githubusercontent.com/31897434/81721095-3fc33c00-9477-11ea-8692-333fafb76ed3.png)

Reply:
I've aadded it in devDependencies. Thanks for the heads up, I'm changing it.